Applications/ProductRatings/res/product_ratings_model.ini

```diff
@@ -16,36 +16,44 @@ epsilon = 1e-7
 
 [input]
 Type = input
-input_shape = 1:2:2 # channel:height:width
+input_shape = 1:1:2 # channel:height:width
 
 [split]
 Type = split
-split_dimension = 2 # split by height
+split_dimension = 3 # split by width
 
-[embedding1]
+[user_embed]
 input_layers = split
 Type = embedding
-in_dim = 15
-out_dim = 8
+in_dim = 6
```

Comment:
add a comment that this must be the max value of the `user id value + 1`.

Reply:
Done